src/OrbitQt/CaptureOptionsDialog.h

```diff
@@ -111,6 +113,7 @@ class CaptureOptionsDialog : public QDialog {
   // `PerfEventOpen.cpp`).
   static constexpr uint16_t kMaxCopyRawStackSizeMaxValue = 65000;
   static constexpr uint16_t kMaxCopyRawStackSizeDefaultValue = 512;
+  static constexpr uint16_t kThreadStateChangeCallstackMaxCopyRawStackSizeDefaultValue = 256;
```

Comment:
Too long. Could you at least remove `Callstack`?

Reply:
Done.